domain/plugins/gui/templates/template.go

```diff
@@ -2,16 +2,52 @@ package templates
 
 import (
 	"context"
+	"html/template"
+	"net/http"
+	"path"
+
 	"github.com/d-velop/dvelop-sdk-go/log"
 	"github.com/shurcooL/httpfs/html/vfstemplate"
-	"html/template"
 )
 
 func parseTemplates() *template.Template {
-	t, e := vfstemplate.ParseGlob(TemplateFileSystem, nil, "*")
+	files, err := listFiles(TemplateFileSystem)
+	if err != nil {
+		panic(err)
+	}
+	t, e := vfstemplate.ParseFiles(TemplateFileSystem, nil, files...)
 	if e != nil {
 		log.Error(context.Background(), e)
 		panic(e)
 	}
 	return t
 }
+
+func listFiles(fs http.FileSystem) ([]string, error) {
```

Comment:
Try to merge `listFiles `and `readDirRecursive `to a single function.
Something like `func getFilesIn(fs http.FileSystem, dir string) ([]string,error) `
and use this function in `pareseTemplates `like `getFilesIn(TemplateFileSystem,"/")`

>

Reply:
Done